test/support/test_evaluator.ex

```diff
@@ -0,0 +1,74 @@
+defmodule TestEvaluator do
+  alias Fika.{
+    Env,
+    TypeChecker,
+    ErlTranslate
+  }
+
+  def eval(code, bindings \\ [])
+
+  # Evaluates a string representing an expression.
+  #
+  # Usage examples:
+  #   1) test "add has less precedence than mult" do
+  #        {7, _} = eval("1 + a * 3", [{:a, "Int", 2}])
+  #      end
+  #
+  #   2) test "match operator" do
+  #        {5, [a: 5]} = eval("a = 5")
+  #      end
+  def eval(str, bindings) when is_binary(str) do
+    case TestParser.expression(str) do
+      {:ok, [parsed], _, _, _, _} ->
+        parsed
+        |> check_types(bindings)
+        |> ErlTranslate.translate_expression()
+        |> eval(Enum.map(bindings, fn {name, _type, value} -> {name, value} end))
+
+      err ->
+        err
+    end
+  end
+
+  # Evaluates a piece of AST representing an expression.
```

Comment:
"Evaluates a piece of AST representing an expression." -> "Evaluates the Erlang abs form of an expression"